[2022-06-06 10:03:08,500] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T07:15:00+00:00 [queued]>
[2022-06-06 10:03:08,508] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T07:15:00+00:00 [queued]>
[2022-06-06 10:03:08,508] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 10:03:08,508] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-06 10:03:08,508] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 10:03:08,706] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): tweets_to_es> on 2022-06-06 07:15:00+00:00
[2022-06-06 10:03:08,710] {standard_task_runner.py:52} INFO - Started process 9310 to run task
[2022-06-06 10:03:08,714] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'tweets_to_es', 'scheduled__2022-06-06T07:15:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/airfow_es.py', '--cfg-path', '/tmp/tmpye4rh379', '--error-file', '/tmp/tmpfoud4qib']
[2022-06-06 10:03:08,715] {standard_task_runner.py:80} INFO - Job 89: Subtask tweets_to_es
[2022-06-06 10:03:08,945] {task_command.py:370} INFO - Running <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T07:15:00+00:00 [running]> on host oury-Tp
[2022-06-06 10:03:09,547] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=tweets_to_es
AIRFLOW_CTX_EXECUTION_DATE=2022-06-06T07:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-06T07:15:00+00:00
[2022-06-06 10:03:09,549] {twitter.py:696} INFO - Retrieving scroll page None
[2022-06-06 10:03:09,549] {twitter.py:626} INFO - Retrieving guest token
[2022-06-06 10:03:09,551] {base.py:171} INFO - Retrieving https://twitter.com/search?f=live&lang=en&q=near%3ATunis+since%3A1654505887+until%3A1654506187&src=spelling_expansion_revert_click
[2022-06-06 10:03:11,502] {base.py:189} INFO - Retrieved https://twitter.com/search?f=live&lang=en&q=near%3ATunis+since%3A1654505887+until%3A1654506187&src=spelling_expansion_revert_click: 200
[2022-06-06 10:03:11,505] {base.py:171} INFO - Retrieving https://api.twitter.com/2/search/adaptive.json?include_profile_interstitial_type=1&include_blocking=1&include_blocked_by=1&include_followed_by=1&include_want_retweets=1&include_mute_edge=1&include_can_dm=1&include_can_media_tag=1&skip_status=1&cards_platform=Web-12&include_cards=1&include_ext_alt_text=true&include_quote_count=true&include_reply_count=1&tweet_mode=extended&include_entities=true&include_user_entities=true&include_ext_media_color=true&include_ext_media_availability=true&send_error_codes=true&simple_quoted_tweets=true&q=near%3ATunis+since%3A1654505887+until%3A1654506187&tweet_search_mode=live&count=100&query_source=spelling_expansion_revert_click&pc=1&spelling_corrections=1&ext=mediaStats%2ChighlightedLabel
[2022-06-06 10:03:12,157] {base.py:189} INFO - Retrieved https://api.twitter.com/2/search/adaptive.json?include_profile_interstitial_type=1&include_blocking=1&include_blocked_by=1&include_followed_by=1&include_want_retweets=1&include_mute_edge=1&include_can_dm=1&include_can_media_tag=1&skip_status=1&cards_platform=Web-12&include_cards=1&include_ext_alt_text=true&include_quote_count=true&include_reply_count=1&tweet_mode=extended&include_entities=true&include_user_entities=true&include_ext_media_color=true&include_ext_media_availability=true&send_error_codes=true&simple_quoted_tweets=true&q=near%3ATunis+since%3A1654505887+until%3A1654506187&tweet_search_mode=live&count=100&query_source=spelling_expansion_revert_click&pc=1&spelling_corrections=1&ext=mediaStats%2ChighlightedLabel: 200
[2022-06-06 10:03:12,589] {logging_mixin.py:115} INFO - {'id': 1533736222480269313, 'username': 'LePapeDiouf', 'tweet_date': datetime.datetime(2022, 6, 6, 9, 3, 5, tzinfo=datetime.timezone.utc), 'tweet_content': '@KalamLecter @blackboxeur77 @Nourdemars3113 @Kulturlesite_ Câ€™Ã©tait dans le volume 2 , un genre parodique de lâ€™Ã©poque, câ€™Ã©tait beaucoup plus soft quâ€™aujourdâ€™hui hein ? ðŸ˜…', 'likes_count': 0, 'tweet_url': 'https://twitter.com/LePapeDiouf/status/1533736222480269313', 'tweet_lang': 'fr'}
[2022-06-06 10:03:12,591] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/oury/airflow/dags/airfow_es.py", line 49, in tweet_to_es_func
    index_es_func(tweets_index, tweet)
TypeError: index_es_func() missing 1 required positional argument: 'tweet'
[2022-06-06 10:03:12,595] {taskinstance.py:1396} INFO - Marking task as FAILED. dag_id=data_pipeline, task_id=tweets_to_es, execution_date=20220606T071500, start_date=20220606T090308, end_date=20220606T090312
[2022-06-06 10:03:12,791] {standard_task_runner.py:92} ERROR - Failed to execute job 89 for task tweets_to_es (index_es_func() missing 1 required positional argument: 'tweet'; 9310)
[2022-06-06 10:03:12,818] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-06 10:03:12,836] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-06 10:42:19,648] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T07:15:00+00:00 [queued]>
[2022-06-06 10:42:19,656] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T07:15:00+00:00 [queued]>
[2022-06-06 10:42:19,656] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 10:42:19,656] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-06 10:42:19,656] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 10:42:19,669] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): tweets_to_es> on 2022-06-06 07:15:00+00:00
[2022-06-06 10:42:19,672] {standard_task_runner.py:52} INFO - Started process 16732 to run task
[2022-06-06 10:42:19,675] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'tweets_to_es', 'scheduled__2022-06-06T07:15:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/airfow_es.py', '--cfg-path', '/tmp/tmpdj05_o8m', '--error-file', '/tmp/tmp3y4twkqf']
[2022-06-06 10:42:19,676] {standard_task_runner.py:80} INFO - Job 89: Subtask tweets_to_es
[2022-06-06 10:42:19,710] {task_command.py:370} INFO - Running <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T07:15:00+00:00 [running]> on host oury-Tp
[2022-06-06 10:42:19,756] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=tweets_to_es
AIRFLOW_CTX_EXECUTION_DATE=2022-06-06T07:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-06T07:15:00+00:00
[2022-06-06 10:42:19,757] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: tweet_to_es_func() missing 5 required positional arguments: 'es', 'start', 'end', 'location', and 'languages'
[2022-06-06 10:42:19,760] {taskinstance.py:1396} INFO - Marking task as FAILED. dag_id=data_pipeline, task_id=tweets_to_es, execution_date=20220606T071500, start_date=20220606T094219, end_date=20220606T094219
[2022-06-06 10:42:19,771] {standard_task_runner.py:92} ERROR - Failed to execute job 89 for task tweets_to_es (tweet_to_es_func() missing 5 required positional arguments: 'es', 'start', 'end', 'location', and 'languages'; 16732)
[2022-06-06 10:42:19,807] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-06 10:42:19,823] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
